data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,450 @@
+			`As you land and exit your ship, you quickly notice you have attracted a lot of attention, probably due to this old design. One Unfettered that looks like an engineer comes to speak to you, warmer than usual. "Welcome. I do not know how you have obtained this ship, although I imagine some of us will see it as a sign. It is a relic of our past, a past we are quite proud of.`
+			`	"As such, most will say that you have no right to use that ship. Thankfully for you, I do not believe in any of that. What matters to me is that this ship is very durable, given the time it obviously spent in the water. It also has ancient technology that would be very valuable to us."`
+			`	They raise their voice for everyone to hear, "I challenge you! The winner will earn the right to own this relic."`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+			`	"No, because in the eyes of my brethren that would mean I did not deserve this ship."`
+			label win
+			`	"You will win the right to access our worlds for just a meagre daily fee of 500 credits. In order to keep our respect, and to not lose on this priceless artifact, this Ancient Ladybug may not be destroyed or sold. You will not be allowed to land so easily as you just did, should you refuse.`
+			`	"Beware, for there are no rules, other than the fact you may only use this one ship. Tell me then, what say you, will you take to the skies, or are you too afraid?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you."`
+					decline
+			`	"Good! I will of course not be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You are not sure if they are being serious or not, but their friends seem to be in for it. Their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" Another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out." Says a third.`
+			label scare
+			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, that this is not a friendly competition between allies. As far as I am concerned, you are an enemy. You should see it more as a fight for you to truly earn this ship, and more, but make no mistakes: it has only two rules: nobody else than us may interfere, and being disabled results in losing the fight. And do not even think about bringing more than this Ladybug as a flagship or we will not bother showing up!"`
+			# this is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles
+			# (but in reality with disabling a lot of ships it goes faster than that)
+			`	"Sometimes the rules can be bent a little. But you certainly did not earn that privilege."`
```

Comment:
I find this a strange shift in his tone of voice and it's a little awkward.

Reply:
```suggestion
			`	"We sometimes will bend the rules for friends, but that is not your case," they say with a look that gives you the chills, before leaving.`
```
smth like this?

Reply:
Well it's supposed to be a tone switch, but I'm open to suggestions
This is a dark warning

Reply:
I guess, so it would be fine if. (though I think `for our friends` is better with the "our")